src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-customtag-update.js

```diff
@@ -124,6 +125,26 @@ export default class EzBtnCustomTagUpdate extends EzWidgetButton {
         );
     }
 
+    renderEzContent() {
+        let value = this.state.content;
+
+        if (value) {
+            value = value.replace(/&lt;/g, '<').replace(/&gt;/g, '>');
+        }
+
+        return (
+            <div className="attribute__wrapper">
+                <label className="attribute__label form-control-label">Content</label>
+                <textarea
+                    className="ezcontent__input form-control"
+                    value={value}
+                    onChange={this.updateContent.bind(this)}
+                />
+            </div>
```

Comment:
AFAIK this is not the good place to render templates, but I'm not familiar with javascripts we use, so I let experts speak;)
ping @sunpietro @tischsoic

Reply:
@alongosz it's React thing. You don't know it ... ;)

Reply:
I have copied the code from equivalent widgets in the same class. If it is not the right place, then it needs to be fixed for all of them.

Reply:
Yeah, so this is why I asked :)
Anyway, nothing here for me to review - all js :P